initServer.sqf

```diff
@@ -1,10 +1,21 @@
 ["Initialize"] call BIS_fnc_dynamicGroups;
 
+//set random time
 _date = date;
 _date set [3,((round(random 10))+8)];
 _date set [4,(round(random 60))];
 setDate _date;
 
+//remove fog, lightning & rain
+0 setFog 0;
+0 setRain 0;
+0 setLightnings 0;
+forceWeatherChange;
+999999 setFog 0;
+999999 setRain 0;
+999999 setLightnings 0;
```

Comment:
waitwhat. 

why is that necessary?

Reply:
damit es auf der Base keines von den drei Sachen gibt!

Reply:
Beißt sich ein bisschen hiermit, oder:

https://github.com/gruppe-adler/Adler_Base_18.Altis/pull/30/files/7be1a07bc8fcb43fc1eb144e8df7779fe469a78a..fc385322daabb00f21360c4306c5d946c2cf0e5c#diff-e32946a6a9180eda8bbf5d43f152033cR18

Das sollten wir vielleicht eh mal rausschmeißen. Kann gut sein, dass es daran liegt, dass die Serverinit teilweise so ewig braucht.

Reply:
Hast du denn eine besser Idee wie man zufälliges wetter und uhrzeit bekommt ohne das es Regen, Nebel Nacht oder Blitze gibt?

Reply:
schräg. 
also:

* `0 setLightnings 0;` setzt das Wetter sofort
* `forceWeatherChange;` zwingt Arma dazu, nich irgendwas random zu tun

soweit versteh ichs.

aber  `999999 setLightnings 0` macht nochmal nen unendlich langsamen Übergang Richtung `lightnings 0`, wo wir doch schon bei lightnings=0 sind. bin verwirrt.

Reply:
Vermute das soll sicherstellen, dass die Editorgesetzten forecasts nicht greifen. Sollten die aber mit manual override eh net imo.

Reply:
Die idee dabei ist das das erst arma sagt mach das jetzt sofort und das zweit verhindert für die Zeit 999999 sec das sich dieser wert ändert. Sonst kann es passieren das du auf einmal bei einer 100% Wolken decke einen Sturm in Arma hast :D

Reply:
Bin der Meinung, die Init war vor dem Wetterchange auch schon ewig lang...